cliquet/initialization.py

```diff
@@ -142,6 +142,9 @@ def on_new_request(event):
             event.request.prefixed_userid = '%s:%s' % (authn_type.lower(),
                                                        user_id)
 
+            # Add a vary Authorization header
+            event.request.response.vary = ['Authorization']
```

Comment:
You are right, we need it for the /v1/ endpoint as well, let me add a test. (because of user_id and bucket_id)